test/utils.tsx

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
 import { v4 } from 'uuid';
 import { renderHook, RenderResult, WaitFor } from '@testing-library/react-hooks';
-import nock from 'nock';
+import nock, { ReplyHeaders } from 'nock';
 import { StatusCodes } from 'http-status-codes';
-import { QueryObserverBaseResult, MutationObserverResult } from 'react-query';
+import { QueryObserverBaseResult, MutationObserverResult } from '@tanstack/react-query';
 import configureHooks from '../src/hooks';
 import { Notifier, QueryClientConfig } from '../src/types';
 import { API_HOST, MOCK_APP_ORIGIN, REQUEST_METHODS } from './constants';
 import configureQueryClient from '../src/queryClient';
 
 type Args = { enableWebsocket?: boolean; notifier?: Notifier; GRAASP_APP_ID?: string | null };
+type NockRequestMethods = Lowercase<`${REQUEST_METHODS}`>;
```

Comment:
I didn't know that `Lowercase<T>` was a thing :sweat_smile:

Reply:
Well I had to add this in order to avoid the TS error related to the index signatures ...